--- origsrc/gnome-subtitles-0.8/Makefile.am	2008-04-14 17:03:51.000000000 -0500
+++ src/gnome-subtitles-0.8/Makefile.am	2008-05-12 06:56:21.921875000 -0500
@@ -3,10 +3,10 @@
 ACLOCAL_AMFLAGS = -I m4
 
 gstpbdir = $(pkglibdir)
-gstpb_SCRIPTS = build/GStreamerPlaybin.dll build/libgstreamer_playbin.so
+gstpb_SCRIPTS = build/GStreamerPlaybin.dll build/cyggstreamer_playbin.dll
 
 $(gstpb_SCRIPTS):
-	cp -f $(GSTPB_DIR)/src/GStreamerPlaybin.dll build && cp -f $(GSTPB_DIR)/src/libgstreamer_playbin.so build
+	cp -f $(GSTPB_DIR)/src/GStreamerPlaybin.dll build && cp -f $(GSTPB_DIR)/src/cyggstreamer_playbin.dll build
 
 EXTRA_DIST = 			\
 	autogen.sh			\
--- origsrc/gnome-subtitles-0.8/build/gnome-subtitles.exe.config	2008-05-11 10:10:45.000000000 -0500
+++ src/gnome-subtitles-0.8/build/gnome-subtitles.exe.config	2008-05-12 08:13:04.796875000 -0500
@@ -1,4 +1,4 @@
 <configuration>
-	<dllmap dll="libenchant" target="libenchant.so.1" />
-	<dllmap dll="libgtkspell" target="libgtkspell.so.0" />
+	<dllmap dll="libenchant" target="cygenchant-1.dll" />
+	<dllmap dll="libgtkspell" target="cyggtkspell-0.dll" />
 </configuration>
--- origsrc/gnome-subtitles-0.8/gstreamer-playbin-0.2.1/src/Makefile.am	2007-12-18 16:31:45.000000000 -0600
+++ src/gnome-subtitles-0.8/gstreamer-playbin-0.2.1/src/Makefile.am	2008-05-11 22:45:02.453125000 -0500
@@ -1,15 +1,13 @@
 AM_CFLAGS = $(gstreamer_CFLAGS) -Wall -g -fPIC
 
-noinst_PROGRAMS = libgstreamer_playbin.so
-libgstreamer_playbin_so_SOURCES = main.c
-libgstreamer_playbin_so_LDFLAGS = $(gstreamer_LIBS) -lgstinterfaces-0.10 -shared -Wl,--as-needed,-soname,libgstreamer_playbin.so
-
-noinst_SCRIPTS = GStreamerPlaybin.dll
+noinst_SCRIPTS = cyggstreamer_playbin.dll GStreamerPlaybin.dll
+cyggstreamer_playbin.dll: main.c
+	$(CC) -shared $(AM_CFLAGS) $(LDFLAGS) -o $@ $^ $(gstreamer_LIBS) -lgstinterfaces-0.10
 
 FILES = Engine.cs AssemblyInfo.cs
 
-GStreamerPlaybin.dll: 
-	$(MCS) -target:library -out:$@ $(FILES)
+GStreamerPlaybin.dll: $(FILES)
+	$(MCS) -target:library -out:$@ $^
 
 EXTRA_DIST = $(FILES) main.c ../README
 CLEANFILES = GStreamerPlaybin.dll
--- origsrc/gnome-subtitles-0.8/src/GnomeSubtitles/Execution/Executable.cs	2007-04-21 21:50:16.000000000 -0500
+++ src/gnome-subtitles-0.8/src/GnomeSubtitles/Execution/Executable.cs	2008-05-12 08:10:39.343750000 -0500
@@ -63,7 +63,6 @@
 
 	public static void Main (string[] args) {
 		Execution execution = new Execution(args);
-		SetProcessName(execution.ExecutableName);
 		Global.Run(execution);
 	}
 
--- origsrc/gnome-subtitles-0.8/src/Makefile.am	2008-04-14 17:10:45.000000000 -0500
+++ src/gnome-subtitles-0.8/src/Makefile.am	2008-05-11 22:45:02.468750000 -0500
@@ -1,8 +1,8 @@
 gnomesubtitlesdir = $(pkglibdir)
 gnomesubtitles_SCRIPTS = $(ASSEMBLY)
 gnomesubtitles_DATA = $(ASSEMBLY_CONFIG)
-GS_BUILDDIR = $(srcdir)/../build
-GS_DATADIR = $(srcdir)/../data
+GS_BUILDDIR = $(top_builddir)/build
+GS_DATADIR = $(top_srcdir)/data
 
 ASSEMBLY = $(GS_BUILDDIR)/gnome-subtitles.exe
 ASSEMBLY_CONFIG = $(ASSEMBLY).config
@@ -49,7 +49,7 @@
 $(ASSEMBLY_CONFIG):
 	cp -f $(srcdir)/GnomeSubtitles/Execution/gnome-subtitles.exe.config $(GS_BUILDDIR)
 
-bin_SCRIPTS = $(srcdir)/GnomeSubtitles/Execution/gnome-subtitles
+bin_SCRIPTS = GnomeSubtitles/Execution/gnome-subtitles
 
 CLEANFILES = $(bin_SCRIPTS) $(ASSEMBLY) $(ASSEMBLY).mdb
 EXTRA_DIST = $(gnomesubtitles_SOURCES) $(GS_SOURCES) $(GS_RESOURCES) $(ASSEMBLY_CONFIG)
